src/MELT/Helpers/LogEntry.cs

```diff
@@ -61,7 +63,13 @@ internal LogEntry(WriteContext entry)
         /// The closer scope for this log entry captured by the current logger.
         /// </summary>
         /// <remarks>This does not track scopes across loggers. For more general scope testing use <see cref="ITestLoggerSink.Scopes"/>.</remarks>
+        [Obsolete("Use FullScope")]
```

Comment:
Following the convention across the library (which is borrower from AspNetCore and related projects)
```suggestion
        [Obsolete("The recommended alternative is " + nameof(FullScope) + ".")]
```